subsys/net/lib/lwm2m_client_utils/Kconfig

```diff
@@ -169,6 +169,9 @@ config LWM2M_CELL_CONN_APN_PROFILE_COUNT
 
 endif #LWM2M_CLIENT_UTILS_CELL_CONN_OBJ_SUPPORT
 
+config LWM2M_CLIENT_UTILS_RAI
+	bool "Enable release assistance indication (RAI)"
+
```

Comment:
This needs extra warnings that we enable REL14 features. It might not be allowed for everyone.

Reply:
However, do not mark them as warnings. We don't use them. Notes are ok.